src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclint/HtmlTag.java

```diff
@@ -509,7 +509,31 @@ public enum Attr {
         VERSION,
         VLINK,
         VSPACE,
-        WIDTH;
+        WIDTH,
+        ACCESSKEY,
```

Comment:
Maybe add a comment `// Global attributes` above this line to indicate the start of global attributes, so future attributes are added to the correct section?

Reply:
Either what Chen has suggested or re-sort the complete list alphabetically as it was prior to this change.

Reply:
> Either what Chen has suggested or re-sort the complete list alphabetically as it was prior to this change.

I will go with Chen's suggestions, will fix in a bit.

I might move STYLE down as it's also a global tag

Reply:
I'm (mostly) with Pavel about `enum` ordering -- it's OK when there is an obvious natural order, but risky when it's not. It's generally better to find other ways to categorize members than relying on the order.